### DISCUSSION TITLE
官网写的貌似不太对

### DISCUSSION THREAD

**Discussion by YSYAdvisable** [apache/doris#27598] [RFC]
![image](https://github.com/apache/doris/assets/82552426/f9a146ef-3b0a-44bf-b9f0-d56c120a8a4d
![1701052593757](https://github.com/apache/doris/assets/82552426/c0a5210b-2d2b-4b30-81d3-1fa30244b515)

**Comment by LemonLiTree** [comment]
It's written backwards. Let me correct it.

**Reply by LemonLiTree** [reply]
https://github.com/apache/doris/pull/27612

**Comment by YSYAdvisable** [comment]
我还提交了一个建议，目前并没得到相应的回复，还有这个官网写错的还没更正，请问是否在下一个发布版本中更正吗？希望给予重视！


杨顺宇
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/doris"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月27日(星期一) 下午2:42
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/doris] 官网写的貌似不太对 (Discussion #27598)





 
#27612

**Comment by LemonLiTree** [comment]
https://github.com/apache/doris/pull/30317
